src/django/api/tests/test_wage_indicator_provider.py

```diff
@@ -0,0 +1,238 @@
+            country_code=self.test_country_code
+        ).delete()
+
+    def test_get_field_name(self):
+        '''Test _get_field_name returns correct field name.'''
+        field_name = self.provider._get_field_name()
+        self.assertEqual(field_name, 'wage_indicator')
+
+    def test_fetch_raw_data_with_valid_country(self):
+        '''Test _fetch_raw_data returns data for valid country code.'''
+        raw_data = self.provider._fetch_raw_data(self.facility)
+        self.assertIsNotNone(raw_data)
+        self.assertEqual(raw_data.country_code, self.test_country_code)
+        self.assertEqual(
+            raw_data.living_wage_link_national,
+            'https://test.example.com/living'
+        )
+
+    def test_fetch_raw_data_with_invalid_country(self):
+        '''Test _fetch_raw_data returns None for non-existent country.'''
+        self.facility.country_code = 'ZZ'
+        raw_data = self.provider._fetch_raw_data(self.facility)
+        self.assertIsNone(raw_data)
+
+    def test_format_data(self):
+        '''Test _format_data formats wage indicator data correctly.'''
+        contributor_info = {
+            'id': self.contributor.id,
+            'name': self.contributor.name,
+            'admin_id': self.contributor.admin_id,
+            'is_verified': self.contributor.is_verified,
+            'contrib_type': self.contributor.contrib_type,
+        }
+
+        formatted_data = self.provider._format_data(
+            self.wage_data,
+            contributor_info
+        )
+
+        self.assertIsInstance(formatted_data, dict)
+        self.assertEqual(formatted_data['field_name'], 'wage_indicator')
+        self.assertEqual(formatted_data['contributor'], contributor_info)
+        self.assertIn('value', formatted_data)
+        self.assertIn('raw_values', formatted_data['value'])
+
+        raw_values = formatted_data['value']['raw_values']
+        self.assertEqual(
+            raw_values['living_wage_link_national'],
+            'https://test.example.com/living'
+        )
+        self.assertIn('living_wage_link_national_text', raw_values)
+
+    def test_fetch_data_complete_flow(self):
+        '''Test complete fetch_data flow returns formatted data.'''
+        data = self.provider.fetch_data(self.facility)
+
+        self.assertIsNotNone(data)
+        self.assertEqual(data['field_name'], 'wage_indicator')
+        self.assertIn('contributor', data)
+        self.assertEqual(data['contributor']['id'], self.contributor.id)
+        self.assertIn('value', data)
+
+    def test_fetch_data_no_wage_data(self):
+        '''Test fetch_data returns None when no wage data exists.'''
+        self.facility.country_code = 'ZZ'
+        data = self.provider.fetch_data(self.facility)
+        self.assertIsNone(data)
+
+    def test_fetch_data_no_contributor_assigned(self):
+        '''Test fetch_data returns None when no contributor assigned.'''
+        # Remove contributor from partner field.
```

Comment:
According to best practices the comments should answer the question `Why` something is being done a certain way, this seems like it's just describing `What` is being done.